cmd/kube-scheduler/app/options/options_test.go

```diff
@@ -1163,7 +1163,7 @@ profiles:
 					}
 					return
 				}
-				assert.NoError(t, err)
+				t.Errorf("unexpected error to create a config: %v", err)
```

Comment:
is there a need to log anything here or are the checks above thorough enough?

for example, if the error here *is* expected, this message is misleading

Reply:
It shouldn't happen here, but if it does, we should emit the error.

Reply:
ah sorry, I misread the code, you are correct